backend/src/main/java/com/gaucho/sos/domain/Story.java

```diff
@@ -0,0 +1,41 @@
+package com.gaucho.sos.domain;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "stories")
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class Story {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @ManyToOne
+    @JoinColumn(nullable = false, name = "user_id")
+    private User user;
+
+    @Column(nullable = false)
+    private String title;
+    @Column(nullable = false)
+    private String shortDescription;
+    @Column(nullable = false)
+    private String longDescription;
+    @Column(nullable = false)
+    private String helpNeeded;
+    @Column(nullable = false)
+    private String postalCode;
+    @Column(nullable = false)
+    private String address;
+
+    @Column(nullable = false)
+    private LocalDateTime createdAt;
+}
```

Comment:
As informações que aparecerão na tela que eu to fazendo são relacionados a esse arquivo?
![Captura de Tela 2024-08-27 às 19 33 25](https://github.com/user-attachments/assets/602791d4-7ae6-405c-91da-9f3d7452fe2c)

Reply:
Sim!

Reply:
Falta: nome do beneficiado, contato, cidade (remover endereço completo).

Reply:
Ajustes dos campos implementados no commit `3bffe4eabd0f5cd5d163687492260e760e194d45`.